src/hooks/useCart/useCart.tsx

```diff
@@ -131,8 +137,10 @@ export const useCart = (
         }
         setQuotaResponse(quotaResponse);
       } catch (e) {
-        // Cart will remain in FETCHING_QUOTA state.
-        if (e instanceof PolicyError) {
+        if (e instanceof EligibilityError) {
```

Comment:
Nit, would it be more straight forward to call this NotEligibleError?

Reply:
agreed haha!!👌